bentoml/_internal/models/model.py

```diff
@@ -75,6 +97,7 @@ def create(
                 metadata=metadata,
                 context=framework_context,
             ),
+            custom_objects,
```

Comment:
should this be part of ModelInfo?

Reply:
I feel like it's best if we leave `ModelInfo` to correspond exactly to the information in the `model.yaml`.

Reply:
yep, this way we can keep the model info lightweight. Custom objects may be heavy